src/payment/strategies/masterpass/masterpass-payment-strategy.ts

```diff
@@ -143,4 +140,37 @@ export default class MasterpassPaymentStrategy implements PaymentStrategy {
         const payload = this._createMasterpassPayload();
         this._masterpassClient.checkout(payload);
     }
+
+    private formatLocale( localeLanguage: string): string {
```

Comment:
Why are you copying and pasting the same function?
i think if you need this function it could be a good idea to added to `src/common/utility` and then re use it everywhere else